versioning/lockfiles.rst

```diff
@@ -220,21 +229,27 @@ We can now proceed iteratively with the following procedure:
         $ cp release/conan.lock build_server_folder/release
         $ cd build_server_folder
 
-4. build the package
+4. clean "modified" nodes from the lockfile
```

Comment:
Not necessary here. You might want to keep the history of modifications to this lockfile at this point, as this node should probably not rebuild things that were already built.

The clean is needed at the end of the iteration, when the whole graph has been built and we want to start a new iteration.